llvm/test/Transforms/MemCpyOpt/memcpy-tbaa.ll

```diff
@@ -0,0 +1,88 @@
+; RUN: opt < %s -passes=memcpyopt,dse -S -verify-memoryssa | FileCheck %s
+; The aim of this test is to check if MemCpyOpt pass merges alias tags
+; after memcpy optimization
+
+; High level overview of this test
+; Input:
+; function test() {
+;   //declaration of local arrays a and b
+;   //initialization of array b in init_loop
+;   //initialization of array a -> copy of array b
+;   //use array a in loop
+; }
+;
+; Expected output after optimization:
+; function test() {
+;   //declaration of local array b
+;   //initialization of array b in init_loop
+;   //use array b in loop
+; }
+
+; ModuleID = 'FIRModule'
+source_filename = "FIRModule"
+target datalayout = "e-m:e-p270:32:32-p271:32:32-p272:64:64-i64:64-i128:128-f80:128-n8:16:32:64-S128"
+target triple = "x86_64-unknown-linux-gnu"
```

Comment:
not needed

Reply:
Thanks for the review, I applied your suggestion.